diff -rup SuperLU_DIST_5.3.0.orig/SRC/get_perm_c_parmetis.c SuperLU_DIST_5.3.0/SRC/get_perm_c_parmetis.c
--- SuperLU_DIST_5.3.0.orig/SRC/get_perm_c_parmetis.c	2019-08-29 10:03:55.495054317 +0100
+++ SuperLU_DIST_5.3.0/SRC/get_perm_c_parmetis.c	2019-08-29 10:05:12.063849626 +0100
@@ -26,7 +26,11 @@ at the top-level directory.
 #include <math.h>
 #include "superlu_dist_config.h"
 #ifdef HAVE_PARMETIS
-#include "parmetis.h"
+#include <stdint.h>
+#include <stdio.h>
+#include "mpi.h"
+#include "ptscotch.h"
+#include "scotchparmetis.h"
 #endif
 #include "superlu_ddefs.h"
 
